src/components/dropdown/Dropdown.tsx

```diff
@@ -192,7 +192,8 @@ export class Dropdown<T = LabelValue> extends React.Component<
             menuPortal: (base: CSSProperties) => ({ ...base, ...menuPortalStyles }),
             valueContainer: (base: CSSProperties) => ({
               ...base, maxHeight: `${maxHeight}px`
-            })
+            }),
+            input: (size === 'small') ? (base: CSSProperties) => ({...base, margin: '0 2px'}) : undefined,
```

Comment:
Why do you need to use some inline style? Could you please use a CSS class instead?

Reply:
No we can't, the issue is explained here https://perzoinc.atlassian.net/browse/APP-4689

Reply:
Ho! I see sorry! ^^